atoma-sui/src/config.rs

```diff
@@ -196,32 +176,180 @@ impl AtomaSuiConfig {
     }
 }
 
+/// Builder for creating Config instances
+pub struct Builder {
```

Comment:
The interface is a nice to have, even though we do not use it, as these structs usually get deserialized from the `config.toml` file (for now). We can keep it though. I would suggest to add proper docs for the struct, including all its fields and methods

Reply:
Good idea!  9db8e55